lib/chef/provider/aws_rds_instance.rb

```diff
@@ -36,14 +36,12 @@ def destroy_aws_object(instance)
     end
     # Wait up to 10 minutes for the db instance to shutdown
     converge_by "waited until RDS instance #{new_resource.name} was deleted" do
-      wait_for(
-        aws_object: instance,
-        query_method: :exists?,
-        expected_responses: [false],
-        acceptable_errors: [AWS::RDS::Errors::DBInstanceNotFound],
-        tries: 60,
-        sleep: 10
-      )
+      tries, sleep = 60, 10
+      Retryable.retryable(:tries => tries, :sleep => sleep, :not => [Aws::RDS::Errors::DBInstanceNotFound]) do |retries, exception|
```

Comment:
All instances of `Aws::` should be `::Aws::` - otherwise we get a `NameError: Uninitialized Constant` error